src/main/webapp/app/modules/conflicts/shared/components/additional-details.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import '../shared-record-view.scss';
 import { connect } from 'react-redux';
+import ScrollToTop from 'react-scroll-up';
```

Comment:
did you added as a new dependency in package.json?

Reply:
Before you wrote about it - no. Now I fixed it. 
I have one more question:
I also added an image of a button "content/images/backToTop_transparent_resized.png". Does it update automatically during commit or I have to add it in a different way?